Source/Core/Core/Movie.cpp

```diff
@@ -643,23 +651,17 @@ static void SetInputDisplayString(ControllerState padState, int controllerID)
 }
 
 // NOTE: CPU Thread
-static void SetWiiInputDisplayString(int remoteID, const u8* const data,
-                                     const WiimoteEmu::ReportFeatures& rptf, int ext,
-                                     const wiimote_key key)
+static void SetWiiInputDisplayString(int remoteID, const DataReportBuilder& rpt, int ext,
+                                     const EncryptionKey& key)
 {
   int controllerID = remoteID + 4;
 
   std::string display_str = StringFromFormat("R%d:", remoteID + 1);
```

Comment:
... oh my god this is doing repeated string appends? aaaaaaaaa

(No Action Required)

Reply:
:P